### DISCUSSION TITLE
Process files based on their names:  No exiftool data but the filename provides information such as YYYMMDDTTTT

### DISCUSSION THREAD

**Discussion by benibilme** [ivandokov/phockup#191] [Q&A] [answered]
Hello,

In my collection, phockup left so many unprocessed pictures that came from various sources, whatsup,  social media etc. which do not  have exifdata, but the filenames consistently have year/mont/day/time structure. Is it possible to phockup to just move and rename files that is extracted from filename by a flag? I have not seen any relevant option in the help message.

Thanks in advance

**Comment by ivandokov** [comment, marked as answer]
Please take a look at this https://github.com/ivandokov/phockup#missing-date-information-in-exif
